web/src/components/onboarding/OnboardingBusinessPersona.tsx

```diff
@@ -25,6 +25,10 @@ export const OnboardingBusinessPersona = <T,>(props: FormContextFieldProps<T>):
     Validate(false);
     setProfileData({
       ...state.profileData,
+      operatingPhase:
+        (event.target.value as BusinessPersona) === "OWNING"
+          ? "GUEST_MODE_OWNING"
+          : state.profileData.operatingPhase,
```

Comment:
[boulder] I think we need additional logic here. Right now, once the user indicates that they have an "up and running" business, they are stuck with a BusinessPersona `GUEST_MODE_OWNING`.

I know we paired on this yesterday, but there's key logic here that we over looked.

I'm logging the persona in this screenshot below. I clicked "starting a business" -> "up and running" -> "starting a business".
<img width="540" alt="Screenshot 2023-08-29 at 3 48 42 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/04720a48-a432-4b4a-a4ad-7804e81b7f96">

Reply:
Additionally, while the end result is that we require the user to set a sector, because we're conditionally setting the `BusinessPersona` we want a test to validate that as well.